installer/msi/Product.wxs

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) 2020-Present MongoDB Inc. -->
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <Product Id="$(var.ProductId)" 
+           Name="!(wix.ProductName)" 
+           Language="1033" 
+           Version="$(var.Version)" 
+           Manufacturer="MongoDB Inc."
+           UpgradeCode="$(var.UpgradeCode)">
+
+	<?if $(var.Platform)=x64 ?>
+            <Package InstallerVersion="301" Compressed="yes" Platform="x64" />
+	<?else ?>
+            <Package InstallerVersion="301" Compressed="yes" Platform="x86"  />
+	<?endif ?>
+
+        <Media Id="1" Cabinet="media1.cab" EmbedCab="yes" />
+        <MajorUpgrade DowngradeErrorMessage="A newer version of [ProductName] is already installed." />
+
+        <WixVariable Id="InstallFolder" Value="$(var.Version)" />
+        <WixVariable Id="ProductName" Value="$(var.ProjectName) $(var.VersionLabel)" />
+
+	<?if $(var.Platform)=x64 ?>
```

Comment:
Since we only support 64-bit windows, I think we should remove these architecture conditionals here and in the other wxs files